packages/@expo/cli/src/run/index.ts

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+import chalk from 'chalk';
+
+import { Command } from '../../bin/cli';
+import { Log } from '../log';
+import { assertWithOptionsArgs } from '../utils/args';
+import { CommandError } from '../utils/errors';
+import { selectAsync } from '../utils/prompts';
+
+export const expoRun: Command = async (argv) => {
+  const args = assertWithOptionsArgs(
+    {
+      // Types
+      '--help': Boolean,
+      // Aliases
+      '-h': '--help',
+      // All other flags are handled by `expoRunAndroid` or `expoRunIos`
+    },
+    {
+      argv,
+      permissive: true,
```

Comment:
If we're doing permissive, then maybe it makes sense to allow a `-p, --platform` flag which can skip the prompt.